modules/apps/frontend-js/frontend-js-importmaps-extender/src/main/java/com/liferay/frontend/js/importmaps/extender/internal/servlet/taglib/JSImportMapsExtenderTopHeadDynamicInclude.java

```diff
@@ -254,11 +359,18 @@ public JSImportMapsRegistration addingService(
 						ServiceReference<JSImportMapsContributor>
 							serviceReference) {
 
+						Long companyId = (Long)serviceReference.getProperty(
+							"com.liferay.importmaps.company");
+
+						if (companyId == null) {
+							companyId = Long.valueOf(_ALL_COMPANIES);
```

Comment:
I believe this case won't happen, there'll be always a companyId, and I believe it's not going to be 0 but whatever id is assigned to the default company. Still, for that case we want the import map only in the default company and not in all of them.

I'd still keep this logic just in case registration does not come with a `com.liferay.importmaps.company`, but unsure whether using `ALL_COMPANIES` or just the default ID

Reply:
No, it can happen because `JSImportMapsContributor` is a public API so maybe someone is using it in their own installation without `companyId`.

To avoid breaking that we must install the contributor for all companies (as it worked before), not for the default company.